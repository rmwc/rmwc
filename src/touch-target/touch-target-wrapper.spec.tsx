import React from 'react';
import { render, screen } from '@testing-library/react';
import { TouchTargetWrapper } from '.';

describe('TouchTargetWrapper', () => {
  it('renders', () => {
    const { asFragment } = render(
      <TouchTargetWrapper>TouchTargetWrapper</TouchTargetWrapper>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
