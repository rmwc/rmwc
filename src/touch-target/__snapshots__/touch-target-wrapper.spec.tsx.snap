// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TouchTargetWrapper renders 1`] = `
<DocumentFragment>
  <div
    class="mdc-touch-target-wrapper"
  >
    TouchTargetWrapper
  </div>
</DocumentFragment>
`;
